rest_api/tests/api_test/payload.py

```diff
@@ -201,7 +256,10 @@ def get_signer():
 def make_intkey_address(name):
     return INTKEY_ADDRESS_PREFIX + hashlib.sha512(
         name.encode('utf-8')).hexdigest()[-64:]
-
+        
+def make_intkey_bad_address(name):
+    return INTKEY_ADDRESS_PREFIX + hashlib.sha512(

```

Comment:
Can there be an another way to create bad address , apart from this ?
Like prefix is incorrect  etc ...
This one seems to return a shortened address